ZenPacks/zenoss/Microsoft/Windows/resources/js/global.js

```diff
@@ -80,7 +87,7 @@ Ext.define("Zenoss.component.WinRMServicePanel", {
     }
 });
 
-ZC.registerName('WinRMService', _t('Service'), _t('Services'));
```

Comment:
I left the .js as-is, since preexisting WinService components will still have their metatype as "WinRMService" prior to the modeler running after upgrade of the ZP.  Not a huge deal, but thought it might alarm users to see that.